3-enrich/scala-common-enrich/project/BuildSettings.scala

```diff
@@ -21,9 +21,9 @@ object BuildSettings {
   // Basic settings for our app
   lazy val basicSettings = Seq[Setting[_]](
     organization          :=  "com.snowplowanalytics",
-    version               :=  "0.21.0",
+    version               :=  "1.24.0", //our Nordstrom version of common enrich
```

Comment:
I was unsure how to version this so I made a major version increment but kept the minor version so we could know which minor version we forked from. I'd love a better suggestion as to how this should be versioned